File: Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python/codeforces_276_B.py
Here is the Python equivalent of the Java code:

```python
def main():
    s = input()
    ar = [0]*26
    
    for i in range(len(s)):
        ch = ord(s[i]) - ord('a')
        ar[ch] += 1
    
    cntod = 0
    for i in range(26):
        if (ar[i] % 2) != 0:
            cntod += 1
    
    if cntod == 0:
        print("First")
    else:
        if cntod % 2 == 0:
            print("Second")
        else:
            print("First")

if __name__ == "__main__":
    main()
```

This Python code does the same thing as the Java code. It reads a string from the user, counts the occurrences of each character in the string, and then determines whether the number of characters with odd counts is even or odd. Based on this, it prints "First" or "Second".

File: Results/DER/Translation/translation_output/no_test/codeqwen/avatar/python/java/codeforces_276_B.java
Here is the Java version of your Python code:

```java
import java.util.HashMap;
import java.util.Map;

public class Main {
    public static void main(String[] args) {
        String s = new Scanner(System.in).nextLine();
        char[] chars = s.toCharArray();
        Map<Character, Integer> dt = new HashMap<>();
        for (char c : chars) {
            if (dt.containsKey(c)) {
                dt.put(c, dt.get(c) + 1);
            } else {
                dt.put(c, 1);
            }
        }
        int cnt = 0;
        boolean x = true;
        for (Map.Entry<Character, Integer> entry : dt.entrySet()) {
            if (entry.getValue() % 2 != 0 && cnt < 1) {
                cnt += 1;
            } else if (entry.getValue() % 2 != 0 && cnt >= 1) {
                x = false;
                break;
            }
        }
        if (x) {
            System.out.println(" First ");
        } else if (!x && s.length() % 2 == 0) {
            System.out.println(" Second ");
        } else if (!x && s.length() % 2 != 0) {
            System.out.println(" First ");
        }
    }
}
```

Please note that this Java code assumes that the input string is provided by the user via the console. If you want to use a different input method, you will need to modify the code accordingly.
